### ISSUE TITLE
Plugin Message channel too large.

### ISSUE THREAD

**Issue by noraemsu** [gisellevonbingen/Minecraft-GravitationalModulatingAdditionalUnit#32]
Channel name - `gravitationalmodulatingunittweaks:gravitationalmodulatingunittweaks`
Running Mohist 517 for 1.20.1

```
Couldn't register custom payload
org.bukkit.plugin.messaging.ChannelNameTooLongException: Attempted to send a Plugin Message to a channel that was too large. The maximum length a channel may be is 64 chars (attempted 67 - 'gravitationalmodulatingunittweaks:gravitationalmodulatingunittweaks.
        at org.bukkit.plugin.messaging.StandardMessenger.validateAndCorrectChannel(StandardMessenger.java:501) ~[forge-1.20.1-47.2.6-universal.jar%23985!/:?]
        at org.bukkit.craftbukkit.v1_20_R1.entity.CraftPlayer.addChannel(CraftPlayer.java:1737) ~[forge-1.20.1-47.2.6-universal.jar%23985!/:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleCustomPayload$16(ServerGamePacketListenerImpl.java:2783) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:1029) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:173) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:1012) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:1006) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:991) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:908) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:281) ~[server-1.20.1-20230612.114412-srg.jar%23980!/:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
        
```

**Comment by gisellevonbingen** [comment]
I will fix this soon

**Comment by gisellevonbingen** [comment]
I think this is fixed in 1.20.1-3.2
https://legacy.curseforge.com/minecraft/mc-mods/gravitational-modulating-additional-unit/files/5027030
